ansible/hosts.ini

```diff
@@ -2,6 +2,8 @@
 host_domain=realraum.at
 ansible_host={{ inventory_hostname }}.{{ host_domain }}
 ansible_user=root
+ansible_port=22000
```

Comment:
if we set this here we should also adapt the ssh snippet generation @ `localconfig` [role](https://github.com/realraum/noc/blob/master/ansible/roles/localconfig/templates/ssh/10r3.conf.j2#L16)

Reply:
Done